production/user_guide.md

```diff
@@ -207,6 +207,58 @@ You will have twenty minutes before your token expires, and you will need to get
 
 Once you’ve downloaded the file, you’ll want to know what to do with the data. We’ve provided a [guide to working with BCDA data](/data-guide/){:target="_self"} to help you, including a crosswalk between CCLF fields and the corresponding sections of the NDJSON files.
 
+
+### Filtering your requests using `_since`
+
+The `_since` parameter lets you filter your bulk data requests by the date when it was updated. This means that instead of receiving all your historical data each time you request data from the API, you will instead be able to enter the date since you last requested data. The API will return data updated between your `_since` input date and the present. The `since` parameter can be used with requests for all resource types.
+
+Before using `_since` for the first time, pull your historical data. Using the `_since` parameter subsequently comprises three steps:
+
+1. Input a date in the correct format.
+2. Start the job to acquire data from an endpoint.
+3. Retrieve data via a job request.
+
+#### a. Pull your historical data.
+
+Before using `_since` for the first time, we recommend that you retrieve all historical data from the BCDA bulk data endpoints (do not use `_since`). Retrieving your historical data before before filtering bulk data with `_since` ensures that there will be no gaps in the claims data delivered from BCDA. 
+
+See [Making Your First Requests for Data](https://bcda.cms.gov/production/user-guide/#making-your-first-requests-for-data) for step-by-step instructions on how to pull your historical data.
+
+After retrieving your historical data, it may be helpful to use the date of your most recent bulk data request for subsequent data pulls using the `_since` parameter. You may retrieve this date by viewing the [_transactionTime_](https://hl7.org/Fhir/uv/bulkdata/export/index.html#response---complete-status) from your last bulk data request.
+
+**Note: Do not input dates before 02-12-2020 into `_since`. Limitations of the Beneficiary FHIR Data (BFD) Server prevent data before 02-12-2020 from being tagged correctly.  For more details, see the [Advanced User Guide](https://bcda.cms.gov/production/technical-user-guide/#filtering-your-data-with-_since).**
```

Comment:
We should not be linking to `https://bcda.cms.gov...` here, because then we can't test it locally. The link should be relative, not absolute. This should be changed to something like `/production/technical-user-guide/#filtering-your-data-with-_since`. There are some examples in this file that you can use as a reference.